sklearn/tests/test_pipeline.py

```diff
@@ -1526,6 +1526,30 @@ def fit(self, X, y):
     check_is_fitted(pipeline)
 
 
+def test_feature_union_check_if_fitted():
```

Comment:
```suggestion
def test_feature_union_check_if_fitted():
    """Check __sklearn_is_fitted__ is defined correctly."""
```

Reply:
Great feedback. Just pushed an update. Tx

Reply:
@thomasjpfan I think linting is currently broken due to a `black` dependency.